### PR TITLE
feature no-video quantums

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

**Comment by jespino** [comment]
De esto falta los scripts de migración, por ahora cierro el pull request.

**Comment by jespino** [comment]
Now included the south migration scripts (model and data)

File: moocng/courses/templates/courses/classroom.html [hunk 1]
```diff
@@ -140,21 +140,12 @@ <h4 class="alert-heading">title</h4>
...
-        }
-    </script>
-    <script src="//www.youtube.com/iframe_api" type="text/javascript"></script>
     <script src="{{ STATIC_URL }}js/libs/bootstrap-fileupload.2.3.1-j6.min.js" type="text/javascript"></script>
     <script src="{{ STATIC_URL }}js/libs/underscore-1.3.3.min.js" type="text/javascript"></script>
     <script src="{{ STATIC_URL }}js/libs/backbone-0.9.2.min.js" type="text/javascript"></script>
-    <script src="{{ STATIC_URL }}js/libs/async.min.js" type="text/javascript"></script>
+    {% media_contents_javascripts %}
     {% compress js %}
+    <script src="{{ STATIC_URL }}js/libs/async.min.js" type="text/javascript"></script>
```
Comment (ablanco) [inline moocng/courses/templates/courses/classroom.html:148]: the async minified file shouldn't be inside the compress tag

**Comment by ablanco** [comment]
I'm getting a 500 error accessing the classroom:

```
ImportError('No module named requests',)
```

I guess requests 1.2.0 is a missing dependence.

File: moocng/media_contents/handlers/__init__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import importlib
+
+from django.conf import settings
+from .null import NullMediaContentHandler
+
+def get_handler(handler_name):
```
Comment (ablanco) [inline moocng/media_contents/handlers/__init__.py:7]: Blank line missing (PEP8)

**Comment by ablanco** [comment]
Some files don't pass PEP8 validation. In this project we follow PEP8 directives but for the "line too long" restriction.

**Comment by ablanco** [comment]
These are the preliminary errors I found in the initial testing of the classroom view and teacher admin:
1. Uncaught TypeError: Cannot call method 'getAttribute' of undefined froogaloop.min.js:2
   - This happens sometimes, even where there are no vimeo videos in the course.
   - The libraries shouldn't be loaded if there aren't nuggets that use them. In a course with only youtube videos, the vimeo library shouldn't be loaded and viceversa.
2. Uncaught ReferenceError: gettext is not defined routes.js:81
   - This happens when I try to access to the root of the classroom view, not directly to a nugget. At least if the module is empty.
3. Moving to the question automatically when the video finishes no longer works, at least with YouTube and Vimeo.
4. I can't navigate to a question nor a peer review.
   - Not with the next button nor directly with url.
   - There are a lot of uncontrolled undefined errors and more problems.
5. YouTube should be the default option when creating a nugget.

@jespino After I merge moocng master into the no-video branch I couldn't use the classroom view (I haven't tried without merging master). Could you fix these problems and send the pull request again? Thanks.

**Comment by jespino** [comment]
Of course :)

**Pull request closed without merging**